src/lib/pages/backoffice/Document/DocumentForms/DocumentEditor/DocumentForm/components/MetadataExtensions.js

```diff
@@ -25,41 +27,98 @@ const getFormComponent = fieldType => {
   }
 };
 
+const sortByLine = (a, b) => {
```

Comment:
what does it do? Maybe a small docstring? I'm not sure by what it is sorted or what is the goal of the sorting?

Reply:
It sorts the components array by the line attribute present in each item, to be able to loop through the array based on the line number and group all the fields that are in the same line (that share same `line` number)

Reply:
See my comment https://github.com/CERNDocumentServer/cds-ils/pull/205#discussion_r517956611